public/js/components/AtomListItem/AtomListItem.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import {atomPropType} from '../../constants/atomPropType.js';
+import {atomTitleExtractor} from '../../util/atomTitleExtractor';
+import publishState from '../../util/publishState';
+import {supportedAtomTypes} from '../../constants/atomData';
+import format from 'date-fns/format';
+import {Link} from 'react-router';
+
+export default class AtomListItem extends React.Component {
+
+    static propTypes = {
+        config: PropTypes.shape({
+            atomEditorUrls: PropTypes.shape({
+                explainer: PropTypes.string,
+                media: PropTypes.string
+            })
+        }),
+        atom: atomPropType
+    };
+
+    getUserString = (changeRecord) => {
+        if (!changeRecord || !changeRecord.user) return 'unknown';
+        if (!changeRecord.user.firstName || !changeRecord.user.lastName) return changeRecord.user.email;
+        return `${changeRecord.user.firstName} ${changeRecord.user.lastName}`;
+    };
+
+    getDateString = (changeRecord) => {
+        if (!changeRecord) return 'unknown';
+        const date = new Date(changeRecord.date);
+        return format(date, 'DD/MM/YYYY HH:mm');
+    };
+
+    renderEditorLink = (atom) => {
+        const title = atomTitleExtractor(atom);
+        if (supportedAtomTypes.map((t)=>t.type).indexOf(atom.atomType) !== -1) {
+            return <Link to={`/atoms/${atom.atomType}/${atom.id}/edit`}
+                         className="atom-list__link atom-list__editor-link"
+                         key={atom.id}>
+                {title}</Link>;
+        }
+        function externalEditorUrl(editorUrls) {
+            switch (atom.atomType) {
```

Comment:
if this gets any larger I think we'll have to split it out into a util. It's fine for now, though